openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -586,7 +586,7 @@ def _convert_and_validate(self, instance, value):
 
 
 class IndexedMappedParameterAttribute(ParameterAttribute):
-    """The attribute of a parameter with an unspecified number of terms, where
+    r"""The attribute of a parameter with an unspecified number of terms, where
```

Comment:
This one also looks fine to me https://openff-docs--1935.org.readthedocs.build/projects/toolkit/en/1935/api/generated/openff.toolkit.typing.engines.smirnoff.parameters.IndexedMappedParameterAttribute.html